frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1517,7 +1517,7 @@ impl<T: Config> ElectionProvider<T::AccountId, T::BlockNumber> for Pallet<T> {
 /// number.
 pub fn dispatch_error_to_invalid(error: DispatchError) -> InvalidTransaction {
 	let error_number = match error {
-		DispatchError::Module { error, .. } => error,
+		DispatchError::Module { error, .. } => error[0],
```

Comment:
Accessing only the most significant byte here is fine in this context, since the pallet error doesn't contain any types and can easily be decoded.

What I'm not sure about is whether this means we should change the container type of `InvalidTransaction::Custom` from `u8` to `[u8; 4]` just to match the current pallet error type. What would be the repercussions be if we change the size of `InvalidTransaction`?